modules/extension/app-schema/app-schema/src/test/java/org/geotools/test/SolrIntegrationTest.java

```diff
@@ -0,0 +1,359 @@
+                        13,
+                        "POINT(8.63 44.92)",
+                        "Alessandria",
+                        Arrays.asList(new String[] {"ALS_TAG_1", "ALS_TAG_2"}),
+                        Arrays.asList(new String[] {"ALS_COMMENT_1", "ALS_COMMENT_2"}));
+        client.add(d2.toSolrDoc());
+
+        StationData d3 =
+                new StationData(
+                        "temperature",
+                        "2016-12-19T11:28:31",
+                        "C",
+                        35,
+                        "BOL",
+                        7,
+                        "POINT(11.34 44.5)",
+                        "Bologna",
+                        null,
+                        null);
+        client.add(d3.toSolrDoc());
+
+        StationData d4 =
+                new StationData(
+                        "temperature",
+                        "2016-12-19T11:28:55",
+                        "C",
+                        25,
+                        "BOL",
+                        7,
+                        "POINT(11.34 44.5)",
+                        "Bologna",
+                        null,
+                        null);
+        client.add(d4.toSolrDoc());
+
+        StationData d5 =
+                new StationData(
+                        "wind speed",
+                        "2016-12-19T11:29:24",
+                        "C",
+                        80,
+                        "BOL",
+                        7,
+                        "POINT(11.34 44.5)",
+                        "Bologna",
+                        null,
+                        null);
+        client.add(d5.toSolrDoc());
+
+        StationData d6 =
+                new StationData(
+                        "pressure",
+                        "2016-12-19T11:30:26",
+                        "hPa",
+                        1019,
+                        "BOL",
+                        7,
+                        "POINT(11.34 44.5)",
+                        "Bologna",
+                        null,
+                        null);
+        client.add(d6.toSolrDoc());
+
+        StationData d7 =
+                new StationData(
+                        "pressure",
+                        "2016-12-19T11:30:51",
+                        "hPa",
+                        1015,
+                        "BOL",
+                        7,
+                        "POINT(11.34 44.5)",
+                        "Bologna",
+                        null,
+                        null);
+        client.add(d7.toSolrDoc());
+
+        client.commit();
+    }
+
+    protected void createField(String name, String type, boolean multiValued) {
+        TestsSolrUtils.createField(client, name, type, multiValued);
+    }
+
+    protected void prepareFiles() throws Exception {
+        // copy meteo.xsd
+        copyTestData("meteo.xsd", testDir);
+
+        // Modify datasource and copy xml
+        File xmlFile =
```

Comment:
This is fine, but another alternative could be putting some placeholder in the file like ``${PLACE_HOLDER}`` and then substitute with a replace all, more easy and cleaner.